torch/csrc/DataLoader.cpp

```diff
@@ -131,6 +131,9 @@ static PyObject *THPModule_errorIfAnyWorkerFails(PyObject *module) {
         std::ostringstream oss;
         oss << "DataLoader worker (pid " << worker_pid << ") is killed "
             << "by signal: " << strsignal(infop.si_status) << ". ";
+        if (infop.si_status == SIGBUS) {
+            oss << "Possibly because out of shared memory. ";
```

Comment:
nit: we can make this a little more detailed. "It is possible that you are out of shared memory. Please try to raise your shared memory limit."